python/graphstorm/eval/evaluator.py

```diff
@@ -109,147 +109,157 @@ def get_val_score_rank(val_score, val_perf_rank_list, comparator):
 
 
 class GSgnnPredictionEvalInterface():
-    """ Interface for prediction evaluation functions
+    """ Interface for Prediction evaluation functions.
 
-    The interface set the two abstract methods for prediction classes, i.e., Classification
-    and Regression, which share the same input arguments.
+    The interface set the two abstract methods for prediction tasks, i.e., **Classification**
+    and **Regression**, which share the same input arguments.
+
+    There are two methdos to be implemented if inherite this interface.
```

Comment:
```suggestion
    There are two methods to be implemented if inheriting this interface.
```

Reply:
revised this part and remove this redundant sentence.